components/searchBox.tsx

```diff
@@ -1,13 +1,55 @@
-import styles from '../styles/Home.module.css';
+// Global
+import { TTailwindString } from 'tailwindcss-classnames';
+import { classnames } from 'tailwindcss-classnames';
 
-export default function SearchBox() {
+type SearchBoxProps = {
+  className?: TTailwindString;
+};
+
+const SearchBox = ({ className }: SearchBoxProps): JSX.Element => {
   return (
-    <div className={styles.container}>
-      <div className={styles.grid}>
-        <div className={styles.searchCard}>
-          <h2>I'm a unified search 🔍</h2>
-        </div>
+    <div className={classnames(className)}>
+      <div className={classnames('relative', 'flex', 'items-center', 'w-full')}>
+        <label className={classnames('sr-only')} htmlFor="scdp-search">
+          Search:
+        </label>
+        <span aria-hidden="true" className={classnames('absolute', 'left-4', 'z-20')}>
+          <svg
+            width="16px"
+            height="16px"
+            viewBox="0 0 16 16"
+            version="1.1"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <title>icon-search</title>
+            <g id="icon-search" fill="currentColor" fillRule="nonzero">
```

Comment:
Unsure if we need the title?

Also the id here and in the path below (Icon-Search) can probably be removed. Could probably remove the `<g>` too